Model/Maker/AbstractMaker.php

```diff
@@ -382,6 +397,21 @@ protected function makeJsMixinFilenameQuestion(string $webArea): Question
         return $question;
     }
 
+    protected function makeGridNamespaceQuestion(
+        \Ctasca\MageBundle\Model\Template\Locator $templateLocator,
+        string $string
+    )
+    {
+        $question = $this->questionFactory->create(
+            'Enter the namespace for the ui component' .
+            "\n<comment>The name of xml will be {{namespace}}_grid \n" .
```

Comment:
The name of xml will be < The name of the xml filename will be {{namespace}}_UICOMPONENT_TYPE (either "form" or "listing")

Reply:
fatto